Assets/Scripts/Networking/ConnectionHandler.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ConnectionHandler : Photon.PunBehaviour {
+
+	[SerializeField] string gameVersion = "0";
+
+	public List<Selectable> connectionReqdObject = new List<Selectable>();
+	List<bool> croStartingStates = new List<bool>();
```

Comment:
If we're going to abbreviate these as cro maybe put in a comment that it stands for "Connection Req(uested?) Object". Or come up with a better name and not abbreviate it?

Reply:
Whichever way you go, they should be the same 🙂 
and the `List<Selectable>` should have a plural name (e.g. `connectionRequestedObjects` )

Reply:
Connection Required Objects

I think the name was carried over from when it wasn't a list. My b.

Reply:
connectionReqdObjectStartStates is an obscene name, btw